### PR TITLE
feat: use "tag" reusable workflow for autobump

### PR DESCRIPTION
**Pull request by aricAI**

### CODE REVIEW

File: .github/workflows/tag.yml [hunk 1]
```diff
@@ -3,23 +3,13 @@ on:
...
-          echo "::set-output name=version::$new_tag"
-      - name: Bump tag
-        uses: mathieudutour/github-tag-action@v6.0
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          custom_tag: ${{ steps.getversion.outputs.version }}
+  tag:
+    uses: AllenNeuralDynamics/aind-reusable-workflows/.github/workflows/tag.yml@main
+    secrets:
+      SERVICE_TOKEN: ${{ secrets.GITHUB_TOKEN }} # required
```
Comment (jtyoung84) [inline .github/workflows/tag.yml:11]: I think this might need to be set to `secrets.SERVICE_TOKEN`?

**Review by jtyoung84 — CHANGES_REQUESTED** [review]
The publishing step might need to be modified to pull in the changes pushed to the main branch. Something like:
```
 publish:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - name: Pull latest changes
        run: git pull origin main
...
```

File: .github/workflows/tag.yml (hunk 1 above)
Comment (aricAI) [reply .github/workflows/tag.yml]: Fixed

**Comment by aricAI** [comment]
Added pull on publish

**Comment by jtyoung84** [comment]
> Added pull on publish

You might need to push the change to the remote repo

**Review by jtyoung84 — APPROVED** [review]

**Pull request merged**